controller/app.controller.js

```diff
@@ -22,4 +22,24 @@ exports.getAllEndpoints = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params
+
+    if (!Number(article_id)){
+        return res.status(400).send({message: "invalid id path"})
```

Comment:
I personally would check for this error in the model and reject the promise if we get an error. This way the error will be piped into our error handling middleware